images/capi/packer/azure/windows-2004.json

```
@@ -0,0 +1,7 @@
+{
+  "build_name": "windows-2004",
+  "image_offer": "WindowsServer",
+  "image_publisher": "MicrosoftWindowsServer",
+  "image_sku": "Datacenter-Core-2004-with-Containers-smalldisk",
+  "vm_size": "Standard_D2s_v3"
+}
```

```suggestion
{
  "build_name": "windows-2004",
  "image_offer": "WindowsServer",
  "image_publisher": "MicrosoftWindowsServer",
  "image_sku": "Datacenter-Core-2004-with-Containers-smalldisk",
  "vm_size": "Standard_D2s_v3",
  "distribution": "windows",
  "distribution_version": "2004"
}
```

I found that if distribution details are user to create image_name in azure-sig.json and are not set for sig-windows builds.  The builds then fail because image_name must match --gallery-image-definition. This is reported in this [issue](https://github.com/hashicorp/packer/issues/7809)

---

good catch. I had validated these at one point but must have updated something.   I thought CI was running them but it looks like it CI calls `build-azure-all` which doesn't include the sig-targets.  

@CecileRobertMichon should `build-azure-all` include `AZURE_BUILD_SIG_TARGETS` the https://github.com/kubernetes-sigs/image-builder/blob/b68517d934d1ff5b2eac21e8cf9f6816f81d8390/images/capi/Makefile#L349